test_pull_requests

```diff
@@ -458,6 +460,22 @@ module Hub
       comment += extended_tests if extended_tests
       recreate_comment(pull_id, bot_comment['id'], repo, comment)
 
+      bot_sha_comment = get_comment_with_prefix(pull_id, repo, BASE_SHA_PREFIX)
+      base_sha_command = %{
```

Comment:
I could be wrong, but I think you're going to want o use the github api to figure this out.  Otherwise it's going to be tough to do from process pull requests.